package.json

```diff
@@ -33,6 +33,7 @@
   "bugs": "http://github.com/linkeddata/rdflib.js/issues",
   "dependencies": {
     "async": "^0.9.x",
+    "growl": "^1.10.5",
```

Comment:
Grown should not be a dependency; it is a transitive dependency.

Instead of doing this, run an `npm audit`, which will give you the right command to update the transitive dependency.

Reply:
Uhm, right... `npm audit` is a feature of `npm` 6?

Reply:
Yes, be sure to be on npm 6 for all Solid projects.